code/modules/mob/mob_grab.dm

```diff
@@ -189,14 +189,14 @@
 		s_click(hud)
 		return
 	if(M == assailant && state >= GRAB_AGGRESSIVE)
-		if( (ishuman(user) && (FAT in user.mutations) && iscarbon(affecting) ) || ( isalien(user) && iscarbon(affecting) ) || ( istype(user,/mob/living/carbon/human/kidan) && istype(affecting,/mob/living/carbon/monkey/diona) ) )
+		if( (ishuman(user) && (FAT in user.mutations) && iscarbon(affecting) ) || ( isalien(user) && iscarbon(affecting) ) || ( istype(user,/mob/living/carbon/human/kidan) && istype(affecting,/mob/living/carbon/monkey/diona) ) || ( ishuman(user) && user.get_species() == "Tajaran"  && istype(affecting,/mob/living/simple_animal/mouse) ) )

```

Comment:
/mob/living/carbon/human/species checks won´t work. The saycode refactor from a while back uses set_species at round start to set the species, rather than spawning an instance of human/species. This means that most mobs will be /mob/living/carbon/human with their species set to something, without having the additional /species at the end (i.e. /kidan). You simple _have_ to check for species.name or get_species.